src/app/pages/dashboard/dashboard.tsx

```diff
@@ -1,10 +1,71 @@
-import { h, Component } from 'preact';
+import {h, Component} from 'preact';
+
+import {TopBar} from "../layouts/top-bar/top-bar";
+import LayoutGrid from "preact-material-components/LayoutGrid";
+import Card, {CardMediaContent} from "preact-material-components/Card";
+import 'preact-material-components/Card/style.css';
```

Comment:
We should be consistent with the quotation marks. Seems like we should use the double one.

Reply:
Done